src/components/UI/ButtonBordered/ButtonBordered.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from './ButtonBordered.module.css';
+
+const button = (props) => (
+    <button
+        disabled={props.disabled}
+        className={[classes.ButtonBodered, classes[props.btnType]].join(' ')}
```

Comment:
classes.ButtonBordered should be instead of classes.ButtonBodered.

Reply:
Done.

Reply:
ok